docker/machine_learning/download/checksums.sha256

```diff
@@ -0,0 +1,4 @@
+fc52756a74d5f371422ff99a7268cb6abb14bfc4a4de3569382cf0bd75e55d36  /data/models/Stable-diffusion/dreamlike-photoreal-2.0.ckpt
```

Comment:
should these be committed?

Reply:
Or should GitHub CI create these

Reply:
These are the checksums for validating the model downloading result. They should be kept.

Reply:
Got it, thanks for the clarification